charts/esphome/0.0.1/values.yaml

```diff
@@ -0,0 +1,63 @@
+# Default values for esphome.
+
+image:
+  repository: esphome/esphome
+  pullPolicy: IfNotPresent
+  tag: 1.15.3
+
+strategy:
+  type: Recreate
+
+service:
+  type: NodePort
+  port:
+    port: 6052
+    nodePort: 36052
+
+env: {}
+  # ESPHOME_DASHBOARD_USE_PING: true
+  # ESPHOME_DASHBOARD_RELATIVE_URL: "/"
+  # ESPHOME_QUICKWIZARD:
+  # ESPHOME_IS_HASSIO:
+  # DISABLE_HA_AUTHENTICATION:
+  # USERNAME:
+  # PASSWORD:
+
+persistence:
+  config:
+    enabled: false
+    emptyDir: false
+    ## Persistent Volume Storage Class
+    ## If defined, storageClassName: <storageClass>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    ##   GKE, AWS & OpenStack)
+    # storageClass: "-"
+    # accessMode: ReadWriteOnce
+    # size: 1Gi
+    ## Do not delete the pvc upon helm uninstall
+    # skipuninstall: false
+    # existingClaim: ""
```

Comment:
Honestly, I want to completely get rid of the upstream common persistance references in due time.

Reply:
agreed, I just left this in to minimize changes to the original.